src/animation/offline/tools/import2ozz_anim.cc

```diff
@@ -385,19 +388,19 @@ bool ImportAnimations(const Json::Value& _config, OzzImporter* _importer,
     }
 
     bool matched = false;
-    for (size_t j = 0; success && j < import_animation_names.size(); ++j) {
+    for (size_t j = 0; j < import_animation_names.size(); ++j) {
       const char* animation_name = import_animation_names[j].c_str();
       if (!strmatch(animation_name, clip_match)) {
         continue;
       }
 
       matched = true;
-      success = ProcessAnimation(*_importer, animation_name, *skeleton,
+      bool anisuccess = ProcessAnimation(*_importer, animation_name, *skeleton,
                                  animation_config, _endianness);
 
       const Json::Value& tracks_config = animation_config["tracks"];
-      for (Json::ArrayIndex t = 0; success && t < tracks_config.size(); ++t) {
-        success = ProcessTracks(*_importer, animation_name, *skeleton,
+      for (Json::ArrayIndex t = 0; anisuccess && t < tracks_config.size(); ++t) {
+        anisuccess = ProcessTracks(*_importer, animation_name, *skeleton,
```

Comment:
If I understand correctly, you don't want to import tracks if animation failed. Right?
But are you sure you want to stop importing further tracks if one failed ?

Reply:
Yeah, I think if one of track import faild, the animation must be bad, and should not play. So I think it should stop the animation import if one of the track faild.